### ISSUE TITLE
Custom catalog tree not exported

### ISSUE THREAD

**Issue by Valentyn-Kubrak** [akeneo/pim-community-dev#3906]
Trying to export product from custom catalog tree. I think you need to add better logs or notifcations.
Because I can't understand what  I need to change to make it works. No errors, no information.

Jus this debug info:

[2016-01-20 19:35:03] batch.DEBUG: Job execution starting: startTime=, endTime=, updatedTime=, status=2, exitStatus=, exitDescription=[], job=[export_navision_products_to_magento] [] []
[2016-01-20 19:35:03] batch.INFO: Step execution starting: id=0, name=[pim_base_connector.jobs.csv_product_export.export.title], status=[2], exitCode=[EXECUTING], exitDescription=[] [] []
[2016-01-20 19:35:04] batch.DEBUG: Step execution success: id= 642 [] []
[2016-01-20 19:35:04] batch.DEBUG: Step execution complete: id=642, name=[pim_base_connector.jobs.csv_product_export.export.title], status=[1], exitCode=[EXECUTING], exitDescription=[] [] []
[2016-01-20 19:35:04] batch.DEBUG: Upgrading JobExecution status: startTime=2016-01-20T18:35:03+00:00, endTime=, updatedTime=, status=3, exitStatus=, exitDescription=[], job=[export_navision_products_to_magento] [] []

Than's all. 

It will be nice if you at least tell me where I need to debug queries. Thanks.

**Comment by Valentyn-Kubrak** [comment]
Can be it related to bug: 
https://github.com/akeneo/pim-community-dev/issues/3686

?

I have integer values for category codes.

**Comment by Valentyn-Kubrak** [comment]
Just completed, No errors, no info. I have right channel with my custom catalog tree for this export, products info completed, products assigned to categories. What I need to do? Please help.

![41de1-clip-30kb](https://cloud.githubusercontent.com/assets/2140144/12459606/5422dc98-bfb8-11e5-93c0-30d1d33e987c.png)

**Comment by nidup** [comment]
Hi @Valentyn-Kubrak 

Yes, agree, we need to improve our import/export log :)

I don't think it's related to the category ids issue.

If you're using the version 1.4, installed with MySQL storage, and exporting via the standard csv product export (at least same reader), you'll be interested by 
 - this reader to see which product ids are loaded https://github.com/akeneo/pim-community-dev/blob/1.4/src/Pim/Bundle/BaseConnectorBundle/Reader/Doctrine/ORMProductReader.php#L237
 - this repository to see which query is executed https://github.com/akeneo/pim-community-dev/blob/1.4/src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/ProductRepository.php#L112

**Comment by Valentyn-Kubrak** [comment]
@nidup 

Thank you for a quick answer. 

After happy night debugging  I found some interesting moment, looks like the problem was in calculation of "Completeness". After deleting  all attributes from my specific "family"  (where assigned these attributes to my channel) export generated records. It's strange because I checked   "Completeness" at least of 10-20 products, but before it did't help.
